lang/ar.json

```diff
@@ -30,5 +30,58 @@
   "This modal can be closed by pressing the Escape key or activating the close button.": "يمكن غلق هذه النافذة المشروطة عن طريق الضغط على زر الخروج أو تفعيل زر الإغلاق",
   ", opens captions settings dialog": ", تفتح نافذة  خيارات التعليقات",
   ", opens subtitles settings dialog": ", تفتح نافذة  خيارات الترجمة",
-  ", selected": ", مختار"
+  ", selected": ", مختار",
+  "Audio Player": "مشغل الصوت",
+  "Video Player": "مشغل الفيديو",
+  "Replay": "إعادة التشغيل",
+  "Seek to live, currently behind live": "ذهاب إلى نقطة البث المباشر، متأخر عن البث المباشر حاليًا",
+  "Seek to live, currently playing live": "ذهاب إلى نقطة البث المباشر، البث المباشر قيد التشغيل حاليًا",
+  "Progress Bar": "شريط التقدم",
+  "progress bar timing: currentTime={1} duration={2}": "توقيت شريط التقدم: currentTime={1} duration={2}",
```

Comment:
This line is incorrectly translated. The translation should read something like `{1} of {2}` or something.

Reply:
Oh, right. 🤔 

I wonder if we should just do `{1} / {2}` for now while we work on getting an updated translation?

Reply:
Might be better to just leave it off for now.